OpenMarket/OpenMarket/Controller/AddProductViewController.swift

```diff
@@ -2,19 +2,162 @@
 
 import UIKit
 
-final class AddProductViewController: UIViewController {
+class AddProductViewController: UIViewController {
+    
+    weak var delegate: UploadDelegate?
+    var productManageView = ProductManageView()
+    let imagePicker = UIImagePickerController()
+    var imageCount = 0
+    var textFieldConstraint: NSLayoutConstraint?
     
     init() {
         super.init(nibName: nil, bundle: nil)
     }
-
+    
     required init?(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         self.view.backgroundColor = .white
+        configureImagePicker()
+        configureNavigationBar()
+        configureDoneButton()
+        configureCancelButton()
+        configureAddImageButton()
+        configureView()
+        initializeHideKeyBoard()
+    }
+    
+    private func configureView() {
+        self.view = productManageView
+        self.productManageView.descriptionTextView.delegate = self
+    }
+    
+    func configureNavigationBar() {
+        let navigationBarAppearance = UINavigationBarAppearance()
+        navigationBarAppearance.backgroundColor = .white
+        navigationBarAppearance.shadowColor = .clear
+        self.navigationItem.title = "상품등록"
+        self.navigationController?.navigationBar.scrollEdgeAppearance = navigationBarAppearance
+    }
+    
+    private func configureDoneButton() {
+        let doneItem = UIBarButtonItem(barButtonSystemItem: .done, target: self,
+                                       action: #selector(doneButtonPressed))
+        self.navigationItem.rightBarButtonItem = doneItem
+    }
+    
+    private func configureCancelButton() {
+        let cancelItem = UIBarButtonItem(barButtonSystemItem: .cancel, target: self,
+                                         action: #selector(cancelButtonPressed))
+        self.navigationItem.leftBarButtonItem = cancelItem
+    }
+    
+    private func configureAddImageButton() {
+        self.productManageView.photoAddButton.addTarget(self, action: #selector(imageAddButtonPressed), for: UIControl.Event.touchUpInside)
```

Comment:
```suggestion
        self.productManageView.photoAddButton.addTarget(self, action: #selector(imageAddButtonPressed), for: .touchUpInside)
```
이미 `for`에는 어떤 타입이 들어갈지 알기 때문에 위와같이 변경하는게 더 좋아보입니다.